corgi/tasks/brew.py

```diff
@@ -897,8 +923,10 @@ def slow_update_brew_tags(build_id: str, tag_added: str = "", tag_removed: str =
             build.meta_attr["tags"] = sorted(tags)
             errata_tag = ADVISORY_REGEX.match(tag_added)
             if errata_tag:
-                # Below should automatically create new relations for this build / erratum
-                slow_load_errata.delay(errata_tag.group())
+                advisory_ids = Brew(BUILD_TYPE).parse_advisory_ids([errata_tag.group()])
```

Comment:
It's a static method, so you don't need to construct an actual Brew instance:
```suggestion
                advisory_ids = Brew.parse_advisory_ids([errata_tag.group()])
                # If there was a result / this erratum was released:
```